modernomad/settings.py

```diff
@@ -15,6 +15,9 @@
     ('Jessy Kate Schingler', 'jessy@embassynetwork.com'),
 )
 
+SECRET_KEY = os.getenv('SECRET_KEY', 'mysecret')
```

Comment:
I would argue this is dangerous, because the app can run in production without a `SECRET_KEY` being set. Perhaps there should be a switch so in development, it falls back to a dummy value, but in production it raises an exception.

FWIW in `settings_docker.py` I have used the `environ` library to set `SECRET_KEY`, which raises an exception if it is not set. If something needs running without a secret key, you can specify it explicitly (see how it is done in `Dockerfile`).

Reply:
I'll just remove this line for the time being.